networking/ovn_kubernetes_network_provider/about-ovn-kubernetes.adoc

```diff
@@ -1,11 +1,56 @@
-[id="learn-about-ovn-kubernetes"]
+[id="about-ovn-kubernetes"]
 = About the OVN-Kubernetes Pod network provider
 include::modules/common-attributes.adoc[]
-:context: learn-about-ovn-kubernetes
+:context: about-ovn-kubernetes
 
 toc::[]
 
-include::modules/nw-ovn-technology-preview.adoc[leveloffset=+1]
+The {product-title} cluster uses a virtualized network for Pod and Service networks.
+The OVN-Kubernetes CNI plug-in is a Pod network provider.
+
+[id="ovn-kubernetes-features"]
+== Features
+
+The OVN-Kubernetes Pod network provider includes the following features:
+
+* Uses Open Virtual Network (OVN) to manage network traffic flows. OVN is a community developed, vendor agnostic network virtualization solution.
+* Implements Kubernetes network policy support, including ingress and egress rules, `podSelector`, and `namespaceSelector`.
```

Comment:
Not sure why `podSelector` and `namespaceSelector` are being called out here. I'd just say "including ingress and egress rules"